pkgs/development/libraries/science/math/openblas/default.nix

```diff
@@ -140,20 +140,6 @@ stdenv.mkDerivation rec {
     sha256 = "1qjr02cqncv20abdp1yzr55n7smhx6h9chqvb0xbp18byynvj87w";
```

Comment:
You need to update the `sha256`, or it will just build the old version.

Reply:
Thank you :facepalm: